src/Castle.Core/DynamicProxy/Generators/Emitters/SimpleAST/ReturnStatement.cs

```diff
@@ -37,10 +38,7 @@ public void Emit(IMemberEmitter member, ILGenerator gen)
 			}
 			else
 			{
-				if (member.ReturnType != typeof(void))
-				{
-					OpCodeUtil.EmitLoadOpCodeForDefaultValueOfType(gen, member.ReturnType);
```

Comment:
Note that this is the only location where the `IMemberEmitter member` parameter gets used (vs. just being passed on to other `Emit` calls).